src/main/java/frc/robot/subsystems/climber/commands/AutoClimb.java

```diff
@@ -0,0 +1,14 @@
+package frc.robot.subsystems.climber.commands;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.subsystems.climber.Climber;
+
+public class AutoClimb extends SequentialCommandGroup {
+
+    private final Climber climber;

```

Comment:
If you don't use this object, why did you instantiated it outside of the constructor